examples/integrations/oci/base/identity-mp/.gitignore

```diff
@@ -0,0 +1 @@
+oci-env
```

Comment:
Please do not include `.gitignore` in example projects. It is only in quickstarts as they serve as templates for CLI and archetypes

Reply:
fixed